### ISSUE TITLE
Preprocessing

### ISSUE THREAD

**Issue by mathematiciseverywhere** [sordonia/hred-qs#9]
Hello,
Good idea. Bu I wonder about preprocess. You wrote on the paper "apply a spelling corrector".  which spell corrector did you use?

**Comment by sordonia** [comment]
Hi,

It's been a long time and I am not sure anymore :(

I think I used something simple like Peter Norvig's spelling corrector.

**Comment by mathematiciseverywhere** [comment]
Hi, 
For example "www.fastandfurioust.com" ---> www fastandfurioust com 
I wonder, Did you delete www and com from sentence?

**Comment by sordonia** [comment]
Yes,

I think I actually remove all queries that contain frequent tokens such as www, http or com, org.
also, i cut fastandfurious into "fast and furious" using an external dictionary

**Comment by mathematiciseverywhere** [comment]
you wrote:
After filtering, we sort the query log by query timestamp and we use the queries submitted before 1 May,
2006 as our background data to estimate the proposed model and the baselines. The next two weeks of data are used as a training set for tuning the ranking models. The remaining two weeks are split into the validation and the test set.

what does mean background data?
